src/genericSchemeRegistry/index.ts

```diff
@@ -12,6 +12,7 @@ const ensRegistryInfo = require("./schemes/ENSRegistry.json");
 const ensPublicResolverInfo = require("./schemes/ENSPublicResolver.json");
 const registryLookupInfo = require("./schemes/RegistryLookup.json");
 const co2kenInfo = require("./schemes/CO2ken.json");
+const OmenDisputeResolutionInfo = require("./schemes/OmenDisputeResolution.json");
```

Comment:
OmenDisputeResolutionInfo  -> omenDisputeResolutionInfo

Reply:
@nicoelzer ?

Reply:
Hey @orenyodfat haven't seen your last comment, just pushed an update that fixes this.